source/mir/ndslice/sparse.d

```diff
@@ -0,0 +1,170 @@
+/++
+$(H2 Sparse Tensors)
+
+This is a submodule of $(LINK2 mir_ndslice.html, mir.ndslice).
+
+
+License:   $(WEB www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+
+Authors:   Ilya Yaroshenko
+
+Source:    $(PHOBOSSRC std/_experimental/_ndslice/_sparse.d)
+
+Macros:
+SUBMODULE = $(LINK2 mir_ndslice_$1.html, mir.ndslice.$1)
+SUBREF = $(LINK2 mir_ndslice_$1.html#.$2, $(TT $2))$(NBSP)
+T2=$(TR $(TDNW $(LREF $1)) $(TD $+))
+T4=$(TR $(TDNW $(LREF $1)) $(TD $2) $(TD $3) $(TD $4))
+
++/
+module mir.ndslice.sparse;
+
+import std.traits;
+import std.meta;
```

Comment:
you always complained about such global imports and advised me to use selective imports.